doc/whatsnew/fragments/8485.false_positive

```diff
@@ -0,0 +1,4 @@
+``invalid-name`` now allows for integers in ``typealias`` names:
+``Good2Name``, ``GoodName2``, ``_1BadName``.
```

Comment:
`_1BadName` is disallowed right?

Reply:
By the pattern, correct. Crafting an updated regex that would allow for `_1SomeName` but not for `1SomeName` would be more complex. At the same time I do not believe it is a good idea to start a variable with a digit even with a `_` or `__` prefix.

If you find the news fragment confusing I can make it more explicit such as:
```
``invalid-name`` now allows for integers in ``typealias`` names:
- now valid: ``Good2Name``, ``GoodName2``.
- still invalid: ``_1BadName``.
```

Reply:
Yes, that suggestion would be perfect! Thanks!

Reply:
Could you still apply this suggestion?

Reply:
Done